WeatherDispatcher/src/main/java/com/example/WeatherDispatcher/model/WeatherCityFetcher.java

```diff
@@ -0,0 +1,43 @@
+package com.example.WeatherDispatcher.model;
+
+import com.example.WeatherDispatcher.utils.TemperatureUtil;
+import lombok.Data;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.RestTemplate;
+import java.time.Instant;
+
+@Data
+@Slf4j
+
+public class WeatherCityFetcher implements Runnable {
+
+    private RestTemplate restTemplate;
+    private String name;
+    private Integer threshold;
+
+    public WeatherCityFetcher(RestTemplate restTemplate, String name, Integer threshold) {
```

Comment:
use lombok annotation instead of defining the constructor, less is more

Reply:
done